docs/cpan-starjacking.md

```diff
@@ -0,0 +1,149 @@
+
+Other ecosystems like PyPI and NPM are also impacted by this problem.
+
+Like other ecosystems, we can attach "metadatas" (repository, website, issue tracker...) to CPAN packages.
+
+It has to be done directly in the release metadatas.
+
+Example of using resources metadata pointing to a popular project, for instance in `META.yml`
+```
+META_MERGE   => {
+    resources => {
+        repository => {
+            type => 'git',
+            url  => 'https://github.com/popular/lot-of-stars.git',
+            web  => 'https://github.com/popular/lot-of-stars',
+        },
+        bugtracker => {
+            web => 'https://github.com/popular/lot-of-stars-and-tickets',
+        },
+    },
+},
+```
+
+Read [CPAN add link to version control system](https://perlmaven.com/how-to-add-link-to-version-control-system-of-a-cpan-distributions) for more about how to set the metadatas.
+
+CPAN metadatas are effectively un-verified.
+
+PyPI now implements a mechanism for verifying metadata with [Project Metadatas Verifications](https://docs.pypi.org/project_metadata/#verified-details):
+
+![PyPI verified](../media/starjacking/pypi-verified.png)
+
+Or it can warn you about metadata information being "unverified":
+
+![PyPI unverified](../media/starjacking/pypi-unverified.png)
+
+Back to (Meta)CPAN, the outcome is that GitHub/Gitlab/... numbers are almost invisible in MetaCPAN. For instance, for a module with more than 200 stars in GitHub:
+
+![GitHub numbers in MetaCPAN](../media/starjacking/metacpan-side.png)
+
+Only issues are visible, but how can issues help in StarJacking?
+
+Still, we can get more details on metadata by hovering **Repository**:
+
+![GitHub Metadatas in CPAN](../media/starjacking/github.png)
+
+As a conclusion, CPAN is not impacted even though CPAN modules can provide inaccurate or incorrect metadata.
+
+It's because GitHub's popularity is not strongly enough propagated to (Meta)CPAN, so lying on metadata (in a particular repository) does not bring any real advantage over other modules.
+
+## PlusPlusJacking ![PlusPlus](../media/starjacking/plusplus.png)
+MetaCPAN package popularity uses an independant indicator, the "++" (PlusPlus).
+
+++ are sent by other CPAN authors.
+
+Manipulation of a PlusPlus score is possible but not cheap: it require creation of several spammy authors.
+
+And creations of authors requires a [request](https://pause.perl.org/pause/query?ACTION=request_id) and [manual acceptance](https://www.nntp.perl.org/group/perl.modules/2024/11/msg105533.html) by CPAN admins.
+
+## RiverJacking ![River](../media/starjacking/river.png)
+The river score appears next to module name in several places:
+
+![River score](../media/starjacking/riverscore.png)
+
+The River can be manipulated, by just making fake modules that depend on the module.
+
+Improving the river score of a module is relatively cheap (possible with one single author with multiple modules).
```

Comment:
possible "at the time of this writing", maybe? I think this is something actionable we can do at the PTS, skip the count on modules by the same author, or think about some sort of "carry" system similar to PageRank. I'll see if Neil is up for talking about this.

Reply:
This is pending and requires follow-up.

Meanwhile, I solved this by removing mentions of anything related to exploiting river jacking.